doc/lua_api.md

```diff
@@ -8039,11 +8041,23 @@ child will follow movement and rotation of that bone.
       * `volumetric_light`: is a table that controls volumetric light (a.k.a. "godrays")
         * `strength`: sets the strength of the volumetric light effect from 0 (off, default) to 1 (strongest)
            * This value has no effect on clients who have the "Volumetric Lighting" or "Bloom" shaders disabled.
+      * `lightIntensity` is a table that controls calculation of night and day sun light color.
```

Comment:
```suggestion
      * `light_intensity` is a table that controls calculation of night and day sun light color.
```

snake_case
Also, what do you mean with night sun light color? Shouldn't be up to the moon during the night? I guess that "calculation of sunlight color" would be also easier to understand